adapters/adnuntius/adnuntius.go

```diff
@@ -233,7 +233,7 @@ func (a *adapter) MakeBids(request *openrtb2.BidRequest, externalRequest *adapte
 func getGDPR(request *openrtb2.BidRequest) (string, string, error) {
 	gdpr := ""
 	var extRegs openrtb_ext.ExtRegs
-	if request.Regs != nil {
+	if request.Regs != nil && request.Regs.Ext != nil {
```

Comment:
I believe the `check-gdpr.json` test covers the case where both `request.Regs != nil` and `request.Regs.Ext != nil` evaluate to true. 
![image](https://user-images.githubusercontent.com/15863840/172205713-b901247f-2937-4c04-b485-b8a165b42805.png)

Can we also add coverage to the scenarios shown below?
| `request.Regs != nil` | `request.Regs.Ext != nil` | `json.Unmarshal(request.Regs.Ext, &extRegs); err != nil` |
|--- | --- | ---|
|false|n/a|n/a|
|true|false|n/a|
|true|true|true|

Reply:
yes, That's true, the only issue is that one of our customers sends an empty regs object, where our adapter returns an error instead of an ad, and they were unable to change that. That's why we need to not return an error. :)

Reply:
any news about this one ?

Reply:
> That's why we need to not return an error. 

Cool. May not return an error if you don't believe you need to. The comment above has to do with adding test coverage to your code. Particularly, adding a test case similar to `adapters/adnuntius/adnuntiustest/supplemental/check-gdpr.json` but with an empty `"regs"` object or an empty `"regs.ext"` so we can test those two scenarios. 

Say a `adapters/adnuntius/adnuntiustest/supplemental/empty-regs.json` file with something like:
```
{
    "mockBidRequest": {
        "id": "test-request-id",
        "user": {
            "id": "1kjh3429kjh295jkl",
            "ext": {
                "consent": "CONSENT_STRING"
            }
        },
        "regs": { },
   .
   .
```
And an additional `adapters/adnuntius/adnuntiustest/supplemental/empty-regs-ext.json` file with something like:
```
{
    "mockBidRequest": {
        "id": "test-request-id",
        "user": {
            "id": "1kjh3429kjh295jkl",
            "ext": {
                "consent": "CONSENT_STRING"
            }
        },
        "regs": {
            "ext": { }
        },
   .
   .
```

Reply:
Sure, will fix it right away!